docs/tutorials/05_problem_transformers.ipynb

```diff
@@ -0,0 +1,461 @@
+   "id": "6cea99d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from qiskit_nature.second_q.operators import ElectronicIntegrals\n",
+    "from qiskit_nature.second_q.problems import ElectronicBasis\n",
+    "from qiskit_nature.second_q.transformers import BasisTransformer\n",
+    "\n",
+    "ao2mo_alpha = np.random.random((2, 2))\n",
+    "ao2mo_beta = np.random.random((2, 2))\n",
+    "\n",
+    "basis_transformer = BasisTransformer(\n",
+    "    ElectronicBasis.AO,\n",
+    "    ElectronicBasis.MO,\n",
+    "    ElectronicIntegrals.from_raw_integrals(ao2mo_alpha, h1_b=ao2mo_beta),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "17922900",
+   "metadata": {},
+   "source": [
+    "## The `FreezeCoreTransformer`\n",
+    "\n",
+    "This transformer provides you with the very simple means to freeze the _core orbitals_ of your molecular system. It requires your problem to contain the `.molecule` attribute from which it can extract the atomic information necessary to perform this Hilbert space reduction."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "4a85694a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Molecule:\n",
+      "\tMultiplicity: 1\n",
+      "\tCharge: 0\n",
+      "\tUnit: Bohr\n",
+      "\tGeometry:\n",
+      "\t\tLi\t(0.0, 0.0, 0.0)\n",
+      "\t\tH\t(0.0, 0.0, 2.8345891868475928)\n",
+      "\tMasses:\n",
+      "\t\tLi\t7\n",
+      "\t\tH\t1\n",
+      "(2, 2)\n",
+      "6\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.drivers import PySCFDriver\n",
+    "\n",
+    "driver = PySCFDriver(atom=\"Li 0 0 0; H 0 0 1.5\")\n",
+    "\n",
+    "full_problem = driver.run()\n",
+    "print(full_problem.molecule)\n",
+    "print(full_problem.num_particles)\n",
+    "print(full_problem.num_spatial_orbitals)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "9a37235a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(1, 1)\n",
+      "5\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.transformers import FreezeCoreTransformer\n",
+    "\n",
+    "fc_transformer = FreezeCoreTransformer()\n",
+    "\n",
+    "fc_problem = fc_transformer.transform(full_problem)\n",
+    "print(fc_problem.num_particles)\n",
+    "print(fc_problem.num_spatial_orbitals)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "92863022",
+   "metadata": {},
+   "source": [
+    "Furthermore, you can provide a list of orbital indices (0-based) which are to be removed from the system.\n",
+    "\n",
+    "> **Note:** these orbitals *must* be unoccupied, otherwise you will incur a large error in your computation. No guards are in place to check that the provided orbital indices are indeed unoccupied, so this is up to you!"
```

Comment:
Even if they are unoccupied it may still be a significant impact dependent on what part, if any, those orbitals play in the dynamics. This we often used with LiH simply because the orbitals in question we would remove were known to play a minimal part - the result was still within chemical accuracy as the energy difference was so small.